docs/reference/modules/express.md

```diff
@@ -0,0 +1,84 @@
+|`trustProxy` | Boolean | Enables the [trust proxy option for Express](https://expressjs.com/en/api.html#trust.proxy.options.table). Set to `true` to tell the Express app to  respect `X-Forwarded-* ` headers. This is helpful when Apostrophe is generating `http:` URLs even though a proxy like nginx is being used to serve it over `https:`. |
+
+### `session`
+
+The `session` options object is passed to the
+[express-session](https://npmjs.org/package/express-session) function. If each is not otherwise specified, Apostrophe enables the following defaults:
+
+```javascript
+{
+  // Does not save sessions until something is stored in them. This greatly
+  // reduces `aposSessions` collection size.
+  saveUninitialized: false,
+  // We are using the 3.x mongo store which is compatible with the
+  // `resave: false` option, preventing the vast majority of session-related
+  // race conditions.
+  resave: false,
+  // Always update the cookie, so that each successive access revives your
+  // login session timeout.
+  rolling: true,
+  // This option should be customized in every project.
+  secret: 'you should have a secret',
+  name: self.apos.shortName + '.sid',
+  cookie: {}
+}
+```
+
+If you want to use another session store, you can pass an instance, but it's easier to let Apostrophe do the work of setting it up:
+
+<AposCodeBlock>
+  ```javascript
+  module.exports = {
+    options: {
+      session: {
+        store: {
+          name: 'connect-redis',
+          options: {
+            // redis-specific options here
+          }
+        }
+      }
+    }
+  }
+  ```
+  <template v-slot:caption>
+    modules/@apostrophecms/express/index.js
+  </template>
+</AposCodeBlock>
+
+Be sure to install `connect-redis`, or the store of your choice, as an npm dependency of your project.
+
+### `csrf`
+
+By default, Apostrophe implements [CSRF protection](https://en.wikipedia.org/wiki/Cross-site_request_forgery) via an `XSRF-TOKEN` cookie. All non-safe HTTP requests (not `GET`, `HEAD`, `OPTIONS` or `TRACE`) automatically receive protection via CSRF middleware, which rejects requests in which the CSRF token does not match the header. If the request was made with a valid API key or bearer token it bypasses this check.
+
+If the `csrf` option is set to `false`, CSRF protection is disabled. **This is not recommended.** Set this option to an object with a `name` property to set that property's value as the CSRF cookie name.
+
+You can configure exceptions to CSRF protection by setting the [`csrfExceptions` option](/reference/module-api/module-options.md#csrfexceptions) of any module to an array of route names specific to that module, or URLs (starting with `/`). Exceptions may use [Minimatch](https://github.com/isaacs/minimatch) wildcards (`*` and `**`).
+
+You may need to use this feature when implementing `POST` form submissions that do not send the header.
```

Comment:
All requests made with an api key or bearer token are exempt from CSRF protection.

Reply:
that's in the first paragraph of this section, but I'll emphasize it.